### PR TITLE
playground popup updated

### PR DESCRIPTION
**Pull request by ritiksaxena124**
**Description**
This PR updates the popup on the layer5.io website

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5a48fd9193fabdf4e2019c4c69a1ac72730e3a8e at: https://64fddc4d00d9d52a2a8e133a--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 4899aabeb1d353a087734b914f781c85eeb30924 at: https://64fe039600d9d544638e14ee--layer5.netlify.app

**Comment by leecalcote** [comment]
Looks great on the iPhone XR
IMG_EE9004426A5B-1

**Review by leecalcote — APPROVED** [review]

File: src/components/Corner-popup/index.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ const CornerPopup = () => {
             <a href="https://play.meshery.io/">
               <img className="popup-image" src={popupImageSmall} />
             </a>
-            <Button primary className="explore-playground-button" title="Explore Meshery Playground" url="https://play.meshery.io/" />
+            <Button primary className="explore-playground-button" title="Explore Playground" url="https://play.meshery.io/" />
```
Comment (leecalcote) [inline src/components/Corner-popup/index.js:31]:
```suggestion
            <Button primary className="explore-playground-button" title="Access Playground" url="https://play.meshery.io/" />
```

**Comment by l5io** [comment]
🚀 Preview for commit 873b9e269a85b4d318d4e1fef24e41cbf7413446 at: https://64fe1231ef5814529ea46ce4--layer5.netlify.app

**Pull request merged**